src/logger/src/metrics.rs

```diff
@@ -349,16 +349,10 @@ pub struct ApiServerMetrics {
 pub struct GetRequestsMetrics {
     /// Number of GETs for getting information on the instance.
     pub instance_info_count: SharedIncMetric,
-    /// Number of failures when obtaining information on the current instance.
-    pub instance_info_fails: SharedIncMetric,
```

Comment:
This one is actually used, here: https://github.com/firecracker-microvm/firecracker/blob/67a83f4ee3b3260c3fe2dcf29ff1c71d589c05da/src/api_server/src/lib.rs#L332
Is this a mistake or do you think we should remove it nevertheless?

Reply:
It was used before, this PR removes that function. With current code it cannot fail, therefore metric is useless.